src/main/java/fi/aalto/cs/apluscourses/model/SubmissionResult.java

```diff
@@ -0,0 +1,103 @@
+package fi.aalto.cs.apluscourses.model;
+
+import static org.apache.commons.lang3.StringUtils.substringBetween;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import org.jetbrains.annotations.NotNull;
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+public class SubmissionResult {
+
+  private int exerciseId;
```

Comment:
Nit: `long` has been used elsewhere for IDs, I'm not sure if it's actually needed but I did it to play it safe